enrichissement.py

```diff
@@ -78,16 +78,44 @@ def main():
 
     df.to_csv("decp_augmente.csv", quoting=csv.QUOTE_NONNUMERIC, sep=";")
 
-
-def detection_accord_cadre(df):
+# Detection accord-cadre
+
+
+def subset_dataframe_annee(df):
+    """En entrée: Dataframe avec une colonne anneeNotification
+    En sortie: une liste de dataframe <- subset de df sur l'annee"""
+    liste_dataframe = []
+    for annee in np.unique(df.anneeNotification):
+        dataframe_annee = df[df.anneeNotification == annee]
+        liste_dataframe += [pd.DataFrame(dataframe_annee)]
+    return liste_dataframe
```

Comment:
un dictionnaire avec pour clé les années serait peut être plus pratique ?

Reply:
Sinon utiliser une list comprehension :
```suggestion
   return [dataframe_annee = df[df.anneeNotification == annee]
              for annee in np.unique(df.anneeNotification)]:
        
```

Reply:
Par pure curiosité, est ce que cette syntaxe à un impact sur le coût de la fonction ? (aussi bien en mémoire qu'en temps ?)